src/main.cpp

```diff
@@ -185,7 +185,8 @@ int main(int argc, const char* argv[]) {
     app.add_flag("--coreneuron", coreneuron_code, "Generate C++ code for CoreNEURON (Default)");
 
     auto host_opt = app.add_subcommand("host", "HOST/CPU code backends")->ignore_case();
-    host_opt->add_flag("--c", c_backend, fmt::format("C++ backend ({})", c_backend))->ignore_case();
+    host_opt->add_flag("--c", cpp_backend, fmt::format("C++ backend ({})", cpp_backend))
```

Comment:
```suggestion
    host_opt->add_flag("--cpp", cpp_backend, fmt::format("C++ backend ({})", cpp_backend))
```

Reply:
oh that's really a nitpick - feel free to ignore.

Reply:
Good suggestion :+1: I have only partly integrated it however because in `NEURON` we pass `host --c` in some place and it would made it incompatible with older `NEURON/CoreNEURON` versions